quartz/components/scripts/explorer.inline.ts

```diff
@@ -101,8 +113,10 @@ function setupExplorer() {
       ) as HTMLElement
 
       // Get corresponding content <ul> tag and set state
-      const folderUL = folderLi.parentElement?.nextElementSibling as HTMLElement
-      setFolderState(folderUL, folderUl.collapsed)
+      const folderUL = folderLi.parentElement?.nextElementSibling
+      if (folderUL) {
+        setFolderState(folderUL as HTMLElement, folderUl.collapsed)
+      }
```

Comment:
this fixes a small bug where errors in console blocked rest of explorer if `localStorage` is in weird state after changes

(technically unrelated to the bug fix of this PR, but didnt want to create a whole new PR to change this one line)